dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -1055,6 +1059,7 @@ export const decidePalette = (format: ArticleFormat): Palette => {
 			matchTitle: textMatchTitle(),
 			byline: textByline(format),
 			twitterHandle: textTwitterHandle(format),
+			twitterHandleLiveBackground: textTwitterHandleLiveBackground(),
```

Comment:
Can we 
1) drop 'background' from the name here? It's already implied by the structure of the object
2) drop 'live' as this is implied by the usage of `format`

Reply:
For this to work you will need to pass in `format`

Reply:
Ahh, I think I see what you are trying to do here. You want to vary the colour based something not contained in `format` like context or a breakpoint?

Reply:
Exactly! I want to vary the colour based on the breakpoint. Do you think `twitterHandleBelowDesktop` would make more sense? And maybe something like:

```suggestion
                         // Call it like that and pass format
			twitterHandleBelowDesktop: textTwitterHandleBelowDesktop(format),
			
			// And in palette 
			const textTwitterHandleBelowDesktop = (format: ArticleFormat): string => {
			        if(format.design === ArticleFormat.Liveblog)
			               return WHITE;
			         
			         return textTwitterHandle(format);      	
                         };
```

What do you think?

Reply:
I pushed a 2nd version of the code. Let me know what you think and if it makes sense!